tests/rptest/tests/delete_records_test.py

```diff
@@ -320,8 +349,8 @@ def obtain_test_parameters(local_snapshot):
             # so conversion must occur to compare
             response = self.admin.get_local_offsets_translated(
                 [truncate_offset], TEST_TOPIC_NAME, 0, translate_to="redpanda")
-            assert len(response) == 1 and 'rp_offset' in response[0], response
-            rp_truncate_offset = response[0]['rp_offset']
+            assert len(response) == 1 and "rp_offset" in response[0], response
+            rp_truncate_offset = response[0]["rp_offset"]
```

Comment:
modifying quotes is not really a necessary change especially in the context of this commit

Reply:
These changes got in by mistake on my part. Will update.